src/dmd/mars.d

```diff
@@ -1853,14 +1863,14 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
         }
         else if (startsWith(p + 1, "verror-style="))
         {
-            const style = p + 1 + "verror-style=".length;
+            const(char)[] style = arg["verror-style=".length + 1 .. $];
 
-            if (strcmp(style, "digitalmars") == 0)
+            if (style == "digitalmars")
                 params.messageStyle = MessageStyle.digitalmars;
-            else if (strcmp(style, "gnu") == 0)
+            else if (style == "gnu")
                 params.messageStyle = MessageStyle.gnu;
             else
-                error("unknown error style '%s', must be 'digitalmars' or 'gnu'", style);
+                error("unknown error style '%s', must be 'digitalmars' or 'gnu'", style.ptr);
```

Comment:
```suggestion
                error("unknown error style '%.*s', must be 'digitalmars' or 'gnu'", cast(int) style.length, style.ptr);
```

Reply:
It's not that easy: `error` is locally redefined in `parseCommandLine` as a function that takes 2 args: https://github.com/dlang/dmd/blob/4601b547a9fde3a03bf38cd2fa13e48670018e0d/src/dmd/mars.d#L1431
Guess I'll have to make this function variadic?

Reply:
Yep, although since it's simply forwarding that won't be too hard. See the other forwarding functions: https://github.com/dlang/dmd/blob/4601b547a9fde3a03bf38cd2fa13e48670018e0d/src/dmd/errors.d#L38-L51

Reply:
Alright, thanks.

Reply:
I'm going with a variadic template now since C-style variadic function requires `extern (C)` or `extern (C++)` linkage, which then makes the `error` function unable to access the `errors` variable defined outside of the function.